### DISCUSSION TITLE
Vertice density for digitized geometries

### DISCUSSION THREAD

**Discussion by sreiber** [opengisch/QField#5270] [Ideas]
It would be very helpful if one can choose the vertice density for digitized objects manually. When people digitize to many geometries with the pen in the contiuously drawing mode, mobile devices will not be able to cope with the current numbers of vertices in areas with alot of geometries.

**Comment by ZsanettMed** [comment]
Hi @sreiber! 
Thank you for sharing your idea. We've moved it to our revitalized [QField Idea Platform](https://ideas.qfield.org/app-feature-requests/p/manual-vertice-density-adjustment), where you and other users can upvote it. This is incredibly helpful for prioritizing our roadmap. Additionally, you have the option to fund this feature, which could speed-up its development. If you're considering this, please reach out to us at [sales@qfield.org](mailto:sales@qfield.org) for more details.

Best,
Zsanett
